### PR TITLE
Homepage CTA Styling Cleanup

### PR DESCRIPTION
**Pull request by dillonmorris91**

### CODE REVIEW

File: src/components/CTASecondary.js [hunk 1]
```diff
@@ -29,35 +28,34 @@ const CTASecondary = ({ classes, className }) => (
...
     display: "flex",
-    justifyContent: "center",
     backgroundColor: "#55C3BA",
     padding: "40px 12px",
-
-    [theme.breakpoints[900]]: {
-      padding: "60px 12px",
-    },
+    justifyContent: "center",
+    color: "#FFF",
```
Comment (chrisheninger) [inline src/components/CTASecondary.js]: I see `#FFF` and `#FFF !important` on some of the selectors below– seems redundant.

I normally expect to see `color` applied directly on the element that contains the text. If we're going to be overriding *all* text in the container, it's fine to have it in root, but then I wouldn't expect it to be repeated on `heading`, `ctaRight`, etc.

File: src/components/CTASecondary.js [hunk 2]
```diff
@@ -29,35 +28,34 @@ const CTASecondary = ({ classes, className }) => (
...
   },
-  ctaContainer: {
+  inner: {
     display: "flex",
+    width: "100%",
+    justifyContent: "center",
```
Comment (chrisheninger) [inline src/components/CTASecondary.js]: This is an extreme nitpick– I try to group similar CSS attributes together.

So in this case, I would expect `display: flex, flexDirection, and justifyContent` to all be next to each other.

Same thing with width/maxWidth, I'd expect those attributes to be next to each other.

File: src/components/CTASecondary.js [hunk 3]
```diff
@@ -29,35 +28,34 @@ const CTASecondary = ({ classes, className }) => (
...
-    justifyContent: "space-between",
-    alignItems: "flex-start",
-    width: "60%",
+    fontWeight: "600",
 
     [theme.breakpoints[900]]: {
       flexDirection: "row",
       textAlign: "left",
       fontSize: "28px",
+      alignItems: "center",
```
Comment (chrisheninger) [inline src/components/CTASecondary.js]: Same nitpick as above– group `alignItems/justifyContent` here with your `flexDirection` attribute

File: src/components/CTASecondary.js (hunk 2 above)
Comment (dillonmorris91) [reply src/components/CTASecondary.js]: I like it! Makes sense to me 👍

**Pull request merged**